Makefile

```diff
@@ -97,7 +97,6 @@ build/nginx-agent:
 build: build/nginx-agent ## Build agent executable
 
 deps: ## Update dependencies in vendor folders
-	git diff --quiet || { echo "Local changes found. Please commit or stash your changes." >&2; exit 1; }
```

Comment:
Was this meant to be removed?

Reply:
yeah, the ```make deps``` shouldn't fail offline, I added this line to the hook separately